Readme.md

```diff
@@ -525,6 +525,21 @@ The changelog can be found in the [Changelog.md](Changelog.md) file.
 
 Firefox doesn’t allow cross-domain fonts: [Specifications](http://www.w3.org/TR/css3-fonts/#font-fetching-requirements), [Bugzilla Ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=604421), [How to fix it](https://coderwall.com/p/v4uwyq).
 
+### My images are getting corrupted
+
+#### Using the node engine
+
+* Certain SVG's are not supported. See the [svg2ttf](https://github.com/fontello/svg2ttf) project which is used to convert from SVG to TTF (which is then converted forward to WOFF and WOFF2).
+* `autoHint` also adjusts the font file anc can cause your font to look different to the SVG so try switchining it off.
```

Comment:
anc = and? But disabling autoHint could also make icons look terrible on Windows.